src/Apps/Artwork/Components/PricingContext.tsx

```diff
@@ -119,10 +118,9 @@ export class PricingContext extends React.Component<PricingContextProps> {
             (bin, index): BarDescriptor => {
               const isFirstBin = index === 0
               const isLastBin = index === artwork.pricingContext.bins.length - 1
-              const binMinPrice = bin.minPrice != null ? bin.minPrice : "$0"
               const title = isLastBin
-                ? `${binMinPrice}+`
-                : `${isFirstBin ? "$0" : binMinPrice}–${bin.maxPrice}`
+                ? `${bin.minPrice}+`
+                : `${isFirstBin ? "$0" : bin.minPrice}–${bin.maxPrice}`
```

Comment:
Is there a way we can leverage metaphysics (or use a library here) to generate the `$` based on the currency of the artwork?

Reply:
Hmm not sure. We get back the currency in the 3-char ISO format so need a way to convert that to a symbol. Seems like a task for another PR, when we make price-in-context multi-currency perhaps.

Reply:
Mind adding a TODO? We're trying to be more pro-active about handling currency 😅 but you're right that price-in-context has a ways to go.

Reply:
done!